test/hooks_test.dart

```diff
@@ -196,5 +197,85 @@ main() {
         });
       });
     });
+
+    group('useContext -', () {
+      DivElement mountNode;
+      _ContextProviderWrapper providerRef;
+      int currentCount = 0;
+      Context<int> testContext;
+      Function useContextTestFunctionComponent;
+
+      setUp(() {
+        mountNode = DivElement();
+
+        UseContextTestComponent(Map props) {
+          final context = useContext(testContext);
+          currentCount = context;
+          return react.div({
+            'key': 'uct1'
+          }, [
+            react.div({'key': 'uct2'}, ['useContext counter value is ${context}']),
+          ]);
+        }
+
+        testContext = react.createContext(1);
+        useContextTestFunctionComponent =
+            react.registerFunctionComponent(UseContextTestComponent, displayName: 'useContextTest');
+
+        react_dom.render(
+            ContextProviderWrapper({
+              'contextToUse': testContext,
+              'mode': 'increment',
+              'ref': (ref) {
+                providerRef = ref;
+              }
+            }, [
+              useContextTestFunctionComponent({'key': 't1'}, []),
+            ]),
+            mountNode);
+      });
+
+      tearDown(() {
+        mountNode = null;
```

Comment:
This still needs an unmount call
```suggestion
        react_dom.unmountComponentAtNode(mountNode);
        mountNode = null;
```

Reply:
🤦‍♂ I'm so used to writing tests in over_react with utils that handle that for me. Sorry for missing that!

Reply:
Right?? No worries!